bindings/2.204/GeometryDash.bro

```diff
@@ -691,29 +697,30 @@ class CCAnimateFrameCache : cocos2d::CCObject {
 class CCBlockLayer : cocos2d::CCLayerColor {
 	// virtual ~CCBlockLayer();
 
-	static CCBlockLayer* create();
+	static CCBlockLayer* create() = win 0x22e40;
 
 	TodoReturn decrementForcePrio();
 	TodoReturn incrementForcePrio();
 
 	virtual bool init() = win 0x22f00;
 	virtual void draw() = win 0x230a0;
-	virtual bool ccTouchBegan(cocos2d::CCTouch*, cocos2d::CCEvent*);
-	virtual void ccTouchMoved(cocos2d::CCTouch*, cocos2d::CCEvent*) {}
-	virtual void ccTouchEnded(cocos2d::CCTouch*, cocos2d::CCEvent*) {}
-	virtual void ccTouchCancelled(cocos2d::CCTouch*, cocos2d::CCEvent*) {}
-	virtual void registerWithTouchDispatcher();
-	virtual void keyBackClicked() = win 0x22ff0;
-	virtual void customSetup() {}
-	virtual TodoReturn enterLayer();
+
+	virtual bool ccTouchBegan(cocos2d::CCTouch*, cocos2d::CCEvent*) = win 0x230d0;
+	virtual void ccTouchMoved(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void ccTouchEnded(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void ccTouchCancelled(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void registerWithTouchDispatcher() = win 0x22fc0;
+	virtual void keyBackClicked();
+	virtual void customSetup();
+	virtual TodoReturn enterLayer() = win 0x22fa0;
 	virtual TodoReturn exitLayer() = win 0x23010;
-	virtual void showLayer(bool) = win 0x23040;
+	virtual TodoReturn showLayer(bool) = win 0x23040;
```

Comment:
why is this changed to TodoReturn

Reply:
probably an merge artifact

Reply:
I didn't really notice that, too bad for me 💀